spec/controllers/api/logs_controller_spec.rb

```diff
@@ -43,4 +40,12 @@
       expect(response.status).to be 403
     end
   end
+
+  def log_read_access(user_id, credential)
+    v = credential.paper_trail.save_with_version
+    v.whodunnit = user_id
+    v.event = :viewed
+    v.created_at = DateTime.now
```

Comment:
This is the way we create read logs on an encryptable (see  [encryptable_controller:135 log_read_access](https://github.com/puzzle/cryptopus/blob/78c73774a59ac000e487ede9e0b7dddf569e5b08/app/controllers/api/encryptables_controller.rb#L135) ) which is why that's how we test it here. Because we reuse this method to generate several read logs I don't think it would make sense to use static dates

Reply:
this way the entries created only differ by a couple of milliseconds. 
I think it's pretty realistic that an encryptable has been access months ago.

Reply:
Got it, changed this for logs and personal logs controller spec